stake-pool/py/stake_pool/actions.py

```diff
@@ -589,3 +589,58 @@ async def decrease_validator_stake(
     signers = [payer, staker] if payer != staker else [payer]
     await client.send_transaction(
         txn, *signers, opts=TxOpts(skip_confirmation=False, preflight_commitment=Confirmed))
+
+
+async def create_token_metadata(client: AsyncClient, payer: Keypair, stake_pool_program_id: PublicKey,
+                                stake_pool_address: PublicKey, token_program_id: PublicKey,
+                                withdraw_authority: PublicKey, name: str, symbol: str, uri: str):
```

Comment:
nit: For the actions, let's keep it simpler, same as you're using it in the test, and in this function you can derive the missing parameters:
```suggestion
async def create_token_metadata(client: AsyncClient, payer: Keypair,
                                stake_pool_address: PublicKey,
                                name: str, symbol: str, uri: str):
```